### PR TITLE
Add cppm tracker markdown

### PR DESCRIPTION
**Pull request by zemerick1**
**Description:**
Added markdown for CPPM device tracker.

**Pull request in [home-assistant](https://github.com/home-assistant/home-assistant) (if applicable):** home-assistant/home-assistant#21673

## Checklist:

- [X ] Branch: `next` is for changes and new documentation that will go public with the next [home-assistant](https://github.com/home-assistant/home-assistant) release. Fixes, changes and adjustments for the current release should be created against `current`.
- [X ] The documentation follows the [standards][standards].

[standards]: https://developers.home-assistant.io/docs/documentation_standards.html

### CODE REVIEW

**Comment by zemerick1** [comment]
I'm not seeing the issue, can someone explain?

File: source/_components/cppm.markdown [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+      hide_if_away: false
+```
+
+{% configuration %}
+host:
+  description: The IP address or hostname of the ClearPass server, e.g., `clearpass.server.com`.
+  required: true
+  type: string
+client_id:
+  description: The client ID from here: `https://clearpass.server.com/guest/api_clients.php`
```
Comment (klaasnicolaas) [inline source/_components/cppm.markdown]:
```suggestion
  description: "The client ID from here: `https://clearpass.server.com/guest/api_clients.php`"
```

File: source/_components/cppm.markdown [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+  required: true
+  type: string
+api_key:
+  description: Secret from here: `https://clearpass.server.com/guest/api_clients.php`
```
Comment (klaasnicolaas) [inline source/_components/cppm.markdown]:
```suggestion
  description: "Secret from here: `https://clearpass.server.com/guest/api_clients.php`"
```

**Review by klaasnicolaas — DISMISSED** [review]

File: source/_components/cppm.markdown [hunk 3]
```diff
@@ -0,0 +1,57 @@
...
+title: "Aruba ClearPass"
+description: "Instructions on how to integrate Aruba routers into Home Assistant."
+date: 2019-03-05 10:45
+sidebar: true
+comments: false
+sharing: true
+footer: true
+logo: aruba.png
+ha_category: Presence Detection
+ha_release: 0.89
```
Comment (MartinHjelmare) [inline source/_components/cppm.markdown]: "0.90"

Keep the quotes to keep the trailing zero.

File: source/_components/cppm.markdown [hunk 4]
```diff
@@ -0,0 +1,57 @@
...
+To use this device tracker in your installation, add the following to your `configuration.yaml` file:
+
+```yaml
+# Example configuration.yaml entry
+device_tracker:
+  - platform: cppm_tracker
+    host: clearpass.server.org
+    client_id: clearpassapi
+    api_key: 00000004qyO513hTdCfjIO2ZWWnmex8QZ5000000000
+    scan_interval: 120
```
Comment (MartinHjelmare) [inline source/_components/cppm.markdown]: Just keep the required config items in the first example. More options can be shown in a second example further below.

File: source/_components/cppm.markdown [hunk 5]
```diff
@@ -0,0 +1,57 @@
...
+description: "Instructions on how to integrate Aruba routers into Home Assistant."
+date: 2019-03-05 10:45
+sidebar: true
+comments: false
+sharing: true
+footer: true
+logo: aruba.png
+ha_category: Presence Detection
+ha_release: 0.89
+---
```
Comment (MartinHjelmare) [inline source/_components/cppm.markdown:14]: Add the `ha_iot_class`.

**Review by MartinHjelmare — DISMISSED** [review]

**Review by frenck — APPROVED** [review]

**Pull request merged**